From 7349de76a9bd8bcb99bc9b8b76f80823ad68f977 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
Date: Mon, 20 May 2019 15:52:07 +0900
Subject: [PATCH 6/8] shared/sec: rename func for parsing PCP

This update is to rename `decode_port_pcp()` to `parse_port_pcp()`.

Signed-off-by: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
---
 src/shared/secondary/spp_worker_th/cmd_parser.c | 17 +++++++++--------
 1 file changed, 9 insertions(+), 8 deletions(-)

diff --git a/src/shared/secondary/spp_worker_th/cmd_parser.c b/src/shared/secondary/spp_worker_th/cmd_parser.c
index 0fd105c..fc30463 100644
--- a/src/shared/secondary/spp_worker_th/cmd_parser.c
+++ b/src/shared/secondary/spp_worker_th/cmd_parser.c
@@ -569,22 +569,23 @@ parse_port_vid(void *output, const char *arg_val,
 	return SPP_RET_OK;
 }
 
-/* decoding procedure of pcp for port command */
+/* Parse PCP for port command */
 static int
-decode_port_pcp(void *output, const char *arg_val,
-				int allow_override __attribute__ ((unused)))
+parse_port_pcp(void *output, const char *arg_val,
+		int allow_override __attribute__ ((unused)))
 {
-	int ret = SPP_RET_OK;
+	int pcp;
 	struct sppwk_cmd_port *port = output;
 	struct spp_port_ability *ability = &port->ability;
 
 	switch (ability->ops) {
 	case SPPWK_PORT_ABL_OPS_ADD_VLANTAG:
-		ret = get_int_in_range(&ability->data.vlantag.pcp,
+		pcp = get_int_in_range(&ability->data.vlantag.pcp,
 				arg_val, 0, SPP_VLAN_PCP_MAX);
-		if (unlikely(ret < SPP_RET_OK)) {
+		if (unlikely(pcp < SPP_RET_OK)) {
 			RTE_LOG(ERR, SPP_COMMAND_PROC,
-					"Bad VLAN PCP. val=%s\n", arg_val);
+					"Invalid `%s`for parsing PCP.\n",
+					arg_val);
 			return SPP_RET_NG;
 		}
 		break;
@@ -863,7 +864,7 @@ cmd_ops_list[][SPPWK_MAX_PARAMS] = {
 		{
 			.name = "port pcp",
 			.offset = offsetof(struct spp_command, spec.port),
-			.func = decode_port_pcp
+			.func = parse_port_pcp
 		},
 		SPPWK_CMD_NO_PARAMS,
 	},
-- 
2.17.1

